src/pattern-library/components/patterns/navigation/TabPage.js

```diff
@@ -0,0 +1,390 @@
+                    {52}
+                  </span>
+                </Tab>
+                <Tab
+                  selected={sidebarPanel === 'pageNotes'}
+                  onClick={() => setSidebarPanel('pageNotes')}
+                >
+                  Page Notes
+                  <span className="relative bottom-[3px] left-[2px] text-[10px]">
+                    {48}
+                  </span>
+                </Tab>
+                <Tab
+                  selected={sidebarPanel === 'orphans'}
+                  onClick={() => setSidebarPanel('orphans')}
+                >
+                  Orphans
+                  <span className="relative bottom-[3px] left-[2px] text-[10px]">
+                    {4}
+                  </span>
+                </Tab>
+              </TabList>
+            </Library.Demo>
+            <p>
+              For tabs that have a simple text label, setting{' '}
+              <code>textContent</code> to that string avoids the jiggle. Text
+              will still change size (bold text is larger) but the tabs
+              themselves do not move.
+            </p>
+            <Library.Demo title="Tabs with textContent (no jiggle)" withSource>
+              <TabList classes="gap-x-6">
+                <Tab
+                  selected={sidebarPanel2 === 'annotations'}
+                  onClick={() => setSidebarPanel2('annotations')}
+                  textContent="Annotations"
+                >
+                  Annotations
+                </Tab>
+                <Tab
+                  selected={sidebarPanel2 === 'pageNotes'}
+                  onClick={() => setSidebarPanel2('pageNotes')}
+                  textContent="Page Notes"
+                >
+                  Page Notes
+                </Tab>
+                <Tab
+                  selected={sidebarPanel2 === 'orphans'}
+                  onClick={() => setSidebarPanel2('orphans')}
+                  textContent="Orphans"
+                >
+                  Orphans
+                </Tab>
+              </TabList>
+            </Library.Demo>
+            <p>
+              For tabs with styled or dynamic content, <code>textContent</code>{' '}
+              can be set to an estimated {'"widest-possible-text-content"'}{' '}
+              value. A trial and error approach worked here:
+            </p>
+            <Library.Demo
+              title="Tabs with estimated widest-value textContent"
+              withSource
+            >
+              <TabList classes="gap-x-6">
+                <Tab
+                  selected={sidebarPanel3 === 'annotations'}
+                  onClick={() => setSidebarPanel3('annotations')}
+                  textContent="Annotations##"
+                >
+                  Annotations
+                  <span className="relative bottom-[3px] left-[2px] text-[10px]">
+                    {52}
+                  </span>
```

Comment:
Do you have plans to create a pattern library component for counts associated with a tab? I suspect this may be a common-enough pattern and have enough considerations around design/a11y that require thought to make it worthwhile.

Reply:
We could do that, sure. It would simplify the acrobatics I did in re `textContent`. I'll give it a think.